modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -6933,22 +7241,6 @@ enum Flamingos2Reads {
 
 }
 
-"""
-Flamingos2 Window Cover
-"""
-enum Flamingos2WindowCover {
-  """
-  Flamingos2WindowCover Open
-  """
-  OPEN
-
-  """
-  Flamingos2WindowCover CLOSE
-  """
-  CLOSE
-
-}
```

Comment:
I didn't see any use of this so I removed it.  We can put it back if it comes up.

Reply:
you may need a breaking-changes label